client_core/src/com/mygdx/game/client_core/ecs/entityfactory/UnitFactory.java

```diff
@@ -44,7 +44,7 @@ private void setUpNameComponent(@NonNull UnitConfig config, int entityId) {
 
   private void setUpPositionComponent(int entityId) {
     Position position = positionMapper.create(entityId);
-    position.getValue().set(0, 0, 0);
+    position.getValue().set(0, 1, 0);
```

Comment:
A tu nie (0,0,0), i zmień w BuildingService na to samo

Reply:
to ważne akurat, bo pomaga przy ustaleniu kolejności, że z tyłu jest Pole, później Jednostka, a nad nimi Highlight. BuildingService nie ma takich rzeczy i nie trzeba tam mieszać